rust/src/treefile.rs

```diff
@@ -1481,6 +1498,12 @@ mod ffi {
         tf.parsed.readonly_executables.unwrap_or(false)
     }
 
+    #[no_mangle]
+    pub extern "C" fn ror_treefile_get_sqlite(tf: *mut Treefile) -> bool {
```

Comment:
I think this would be clearer as just a getter for the `rpmdb` field. So e.g. `ror_treefile_get_rpmdb` and return a `*const libc::c_char` (which would be `NULL` if not provided).

That way we avoid the ambiguous meaning of `false` and clearly encode the default in only one spot (in `rpmostree-core.c`).